CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Next release
+
+- [CL-1058] - Add desktop preview
+
```

Comment:
Wondering if this should be a bit more descriptive for Product/GovSuccess? (Maybe just adding "in content builder" to the end?)